openassessment/assessment/models/training.py

```diff
@@ -137,7 +137,7 @@ def calculate_hash(answer, options_selected, rubric):
             'options_selected': options_selected,
             'rubric': rubric.id
         })
-        return sha1(contents).hexdigest()
+        return sha1(contents.encode('utf-8')).hexdigest()
```

Comment:
Although this change not require for bom-415 but i did it to avoid any future issue. Please let me know if not require.